### DISCUSSION TITLE
Warning when using snapshot within contextApi

### DISCUSSION THREAD

**Discussion by dioggosoares83** [pmndrs/valtio#916] [Bug report]
### Bug Description

![image](https://github.com/pmndrs/valtio/assets/170574231/98f73207-9b31-4eb9-b5d0-eb196fdc6110)

version: 1.13.2

I'm trying to use valtio in the context, but when I call the snapshot and return it within the hook, the warning appears and the component has a delay in rendering

### Check List

- [X] I've added a link to a [TypeScript Playground](https://www.typescriptlang.org/play), a [StackBlitz Project](https://stackblitz.com/) or something else with a minimal reproduction.

**Comment by dai-shi** [comment]
I'm not sure, but it looks like it's app code using context api, not Valtio's code.

**Reply by dioggosoares** [reply]
Yes, valtio inside contextapi, https://valtio.pmnd.rs/docs/how-tos/how-to-use-with-context

**Reply by dai-shi** [reply]
Can you please create a very small reproduction with StackBlitz?
